rllib/models/tf/tf_action_dist.py

```diff
@@ -64,32 +64,25 @@ def deterministic_sample(self) -> TensorType:
 
     @override(ActionDistribution)
     def logp(self, x: TensorType) -> TensorType:
-        return -tf.nn.sparse_softmax_cross_entropy_with_logits(
-            logits=self.inputs, labels=tf.cast(x, tf.int32)
-        )
+        logp = tf.math.log(tf.nn.softmax(self.inputs))
```

Comment:
Why don't we clip by value with tf.keras.backend.epsilon() here, too?

Reply:
Other than that, sparse_softmax_cross_entropy_with_logits also does the job here. So I don't think a change is needed.